demos/modals/ModalAuthentication.tsx

```diff
@@ -121,10 +123,22 @@ const ModalAuthentication: ModalComponent<ModalAuthenticationProps> = (props) =>
             </Button>
           )}
           {loading ? null : (
-            <Button visual loading={loading} style={{ marginTop: 16, width: '100%' }}>
+            <Button onClick={() => setShowHandleInput(true)} style={{ marginTop: 16, width: '100%' }}>
               <Bluesky height="16px" style={{ marginRight: 12, color: '#0A7AFF' }} /> Sign in with Bluesky
             </Button>
           )}
+          {showHandleInput && (
+            <div style={{ marginTop: 24 }}>
+              <InputLabel>Enter your Bluesky handle:</InputLabel>
+              <Input value={blueskyHandle} onChange={(e) => setBlueskyHandle(e.target.value)} placeholder="e.g., @yourhandle.bsky.social" style={{ width: '100%', marginTop: 8 }} />
```

Comment:
Can you double check if the `100%` is necessary? I think Input Fields are fluid and full width by default. I can't remember.

Reply:
No it's not fluid by default.